fdo.patch

```diff
@@ -0,0 +1,92 @@
+diff --git a/garglk/CMakeLists.txt b/garglk/CMakeLists.txt
+index 2d9b935e..4dd4167d 100644
+--- a/garglk/CMakeLists.txt
++++ b/garglk/CMakeLists.txt
+@@ -239,9 +239,9 @@ elseif(UNIX)
+     endif()
+ 
+     if(WITH_FREEDESKTOP)
+-        install(FILES "gargoyle.desktop" DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/applications")
+-        install(FILES "gargoyle-house.png" DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/icons")
```

Comment:
I would just make it install it in the right place

Reply:
I'm not entirely sure what you mean.  This is a low-resolution PNG that doesn't satisfy the requirements for Flathub, so it's not installed here, and instead a replacement SVG is being installed.  At some point we'll switch the official icon to an SVG, but the current release just doesn't have one.